src/popup/router/pages/SecuritySettings.vue

```diff
@@ -97,7 +97,7 @@ export default {
   methods: {
     setBackedUpSeed() {
       this.$store.commit('setBackedUpSeed');
-      this.$router.push({ name: 'account' });
+      this.$store.dispatch('router/back');
```

Comment:
```suggestion
      this.$store.dispatch('router/back');
```
This is not a good idea. If you will open extension on this page, than this button will lead you to an account page.

Reply:
Then 9ada9dd